wp-content/plugins/revisionary/admin/admin_rvy.php

```diff
@@ -0,0 +1,899 @@
+			jQuery(document).ready( function($) {
+				$('#rvy-rev-checkall').click(function() {
+					$('.rvy-rev-chk').attr( 'checked', this.checked );
+				});
+			});
+			/* ]]> */
+			</script>
+			<?php
+			wp_enqueue_script( 'rvy_edit', RVY_URLPATH . "/admin/revision-edit.js", array('jquery'), RVY_VERSION, true );
+			
+			if ( ( empty( $_GET['action'] ) || in_array( $_GET['action'], array( 'view', 'edit' ) ) ) && ! empty( $_GET['revision'] ) ) {
+				if ( $revision = get_post( $_GET['revision'] ) ) {
+					if ( ( 'revision' != $revision->post_type ) || $post = get_post( $revision->post_parent ) ) {
+				
+						// determine if tinymce textarea should be editable for displayed revision
+						global $current_user;
+
+						if ( 'revision' != $revision->post_type ) // we retrieved the parent (current revision) that corresponds to requested revision
+							$read_only = true;
+
+						elseif ( ( 'rvy-pending' == $revision->post_status ) && ( $revision->post_author == $current_user->ID ) )
+							$read_only = false;
+						else {
+							if ( $type_obj = get_post_type_object( $post->post_type ) )
+								$read_only = ! current_user_can( $type_obj->cap->edit_post, $revision->post_parent );
+						}
+
+						$this->tinymce_readonly = $read_only;
+						
+						require_once( dirname(__FILE__).'/revision-ui_rvy.php' );
+						
+						add_filter( 'tiny_mce_before_init', 'rvy_log_tiny_mce_params', 1 );
+						add_filter( 'tiny_mce_before_init', 'rvy_tiny_mce_params', 998 );	// this is only applied to revisionary admin URLs, so not shy about dropping the millennial hammer
+
+						if ( $read_only )
+							add_filter( 'tiny_mce_before_init', 'rvy_tiny_mce_readonly', 999 );
+						
+						// WP Super Edit Workaround - $wp_super_edit->is_tinymce property is currently set true only if URI matches unfilterable list: '/tiny_mce_config\.php|page-new\.php|page\.php|post-new\.php|post\.php/'
+						global $wp_super_edit;
+						
+						if ( ! empty($wp_super_edit) && ! $wp_super_edit->is_tinymce )
+							include_once( dirname(__FILE__).'/super-edit-helper_rvy.php' );
+						//
+						
+						if ( ! awp_ver( '3.3-dev' ) )
+							wp_tiny_mce();
+					}
+				}
+			}
+			
+			// need this for editor swap from visual to html
+			if ( empty($read_only) )
+				wp_print_scripts( 'editor', 'quicktags' );
+			else {
+				wp_print_scripts( 'editor' );
+				
+// if the revision is read-only, also disable the HTML editing area and kill the toolbar which the_editor() forces in
+?>
+<script type="text/javascript">
+/* <![CDATA[ */
+jQuery(document).ready( function($) {
+	$('#ed_toolbar').hide();
+	$('#content').attr('disabled', 'disabled');
+});
+/* ]]> */
+</script>
+<?php	
+			} // endif read_only
+
+			require_once( dirname(__FILE__).'/revision-ui_rvy.php' );
+
+			rvy_revisions_js();
+		}
+		
+		// all required JS functions are present in Role Scoper JS; TODO: review this for future version changes as necessary
+		// TODO: replace some of this JS with equivalent JQuery

```

Comment:
noch zu erledigen? jQuery in WordPress muss man tatsächlich über `jQuery` statt `$` machen.